domain/service/service.go

```diff
@@ -308,6 +332,22 @@ func (s *Service) AddVirtualHost(application, vhostName string) error {
 	return fmt.Errorf("unable to find application %s in service: %s", application, s.Name)
 }
 
+// EnableVirtualHost enable or disable a virtual host for given service
+func (s *Service) EnableVirtualHost(application, vhostName string, enable bool) error {
```

Comment:
To help debug cases with invalid input, it would be helpful to return errors that identify undefined application and vhost names